lib/avromatic/rspec.rb

```diff
@@ -0,0 +1,9 @@
+require 'webmock/rspec'
+require 'avro_turf/test/fake_schema_registry_server'
+
+RSpec.configure do |config|
+  config.before(:each) do
+    WebMock.stub_request(:any, /^#{Avromatic.registry_url}/).to_rack(FakeSchemaRegistryServer)
+    FakeSchemaRegistryServer.clear
+  end
+end
```

Comment:
We talked it through and we're going to leave this as-is for now and split it out later if we add more test utilities.